test/ERC721BridgelessMinting.ts

```diff
@@ -36,6 +36,49 @@ describe("ERC721BridgelessMinting", function () {
     await erc721.waitForDeployment();
   });
 
+  it("Should support the standard ERC721 interface", async function () {
+    const InterfaceIdFactory = await ethers.getContractFactory(
+      "InterfaceId",
+    );
+    const interfaceId = await InterfaceIdFactory.deploy();
+    await interfaceId.waitForDeployment();
+
+    // Tests both via direct contract calls, as well the on-chain OpenZeppelin lib checker:
+    const specified721Id = "0x80ac58cd";
+    expect(await interfaceId.getERC721Id()).to.equal(specified721Id);
+    expect(await erc721.supportsInterface(specified721Id)).to.equal(true);
+    expect(await interfaceId.supportsInterface(await erc721.getAddress(), specified721Id)).to.equal(true);
+  });
+
+  it("Should support standard ERC165 interface", async function () {
+    const InterfaceIdFactory = await ethers.getContractFactory(
+      "InterfaceId",
+    );
+    const interfaceId = await InterfaceIdFactory.deploy();
+    await interfaceId.waitForDeployment();
+
+    // Tests both via direct contract calls, as well the on-chain OpenZeppelin lib checker:
+    const mustReplyTrue = "0x01ffc9a7";
+    const mustReplyFalse = "0xffffffff";
+    expect(await erc721.supportsInterface(mustReplyTrue)).to.equal(true);
+    expect(await erc721.supportsInterface(mustReplyFalse)).to.equal(false);
+    expect(await interfaceId.supportsERC165(await erc721.getAddress())).to.equal(true);
+  });
+
+  it("Should support bridgeless minting ERC721 interface", async function () {
```

Comment:
I would add a negative test to make sure that supportsInterface not always returns true

Reply:
there is already one in line 62